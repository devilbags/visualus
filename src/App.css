.logo {
  width: 50px;
  height: 31px;
  background: rgba(255, 255, 255, 0.2);
  margin: 16px 24px 16px 0;
  float: left;
}

.ant-layout-content {
  margin-top: 64px; /* Because header is fixed on top with line height of 64px */
}
