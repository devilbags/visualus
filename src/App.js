import React from 'react';

class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      title: 'Custom Title'
    };
  }

  render() {
    return (
      <div className="App">
        <h1>Title</h1>
      </div>
    );
  }
}

export default App;
