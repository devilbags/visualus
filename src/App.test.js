import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
import { lstat } from 'fs';

it('App renders without crashing', () => {
  shallow(<App />);
});
