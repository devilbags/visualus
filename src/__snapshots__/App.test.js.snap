// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matches snapshot 1`] = `
<Adapter>
  <Adapter
    style={
      Object {
        "position": "fixed",
        "width": "100%",
      }
    }
  >
    <span
      className="logo"
    />
    <Menu
      className=""
      defaultSelectedKeys={
        Array [
          "1",
        ]
      }
      mode="horizontal"
      prefixCls="ant-menu"
      style={
        Object {
          "lineHeight": "64px",
        }
      }
      theme="dark"
    >
      <MenuItem>
        <Link
          replace={false}
          to="/"
        >
          Home
        </Link>
      </MenuItem>
      <MenuItem>
        <Link
          replace={false}
          to="/education"
        >
          Education
        </Link>
      </MenuItem>
    </Menu>
  </Adapter>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/education"
  />
  <Adapter
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    VisualUSA Â©2017 Created by Joey Liu
  </Adapter>
</Adapter>
`;
